packages/clarity-decode/types/interaction.d.ts

```diff
@@ -3,13 +3,18 @@ import { PartialEvent } from "./core";
 
 export interface ClickData extends Interaction.ClickData {
     hashBeta: string;
+    checksum?: string;
 }
 
 export interface TimelineData extends Interaction.TimelineData {
     hashBeta: string;
 }
 
-export interface InputEvent extends PartialEvent { data: Interaction.InputData; }
+export interface InputData extends Interaction.InputData {
+    checksum?: string;
```

Comment:
Why not put this field in the base type?

Reply:
Because it's part of the value (where possible) and not a separate field.

Reply:
Sorry, don't understand the response. Can you clarify. Why only needed in these two cases?

Reply:
Cleaned it up -- not part of the base type.